### PR TITLE
Update README for usability. 

### PR DESCRIPTION
**Pull request by BenRKarl**
All documentation will be consolidated on devsite at: https://developers.google.com/google-ads/api/docs/client-libs/python

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -13,213 +13,10 @@ Features
 * Easy management of credentials.
 * Easy creation of Google Ads API service clients.
 
-Requirements
-------------
-* Python 3.7+
```
Comment (mcloonan) [inline README.rst:18]: You can keep this section. Quick reference info that is useful at a glance can be useful in the GitHub repo.

Comment (BenRKarl) [reply README.rst:18]: Sounds good, in that case I think I'll add the one-line installation instructions too because that's very useful to have here.

**Comment by wihl** [comment]
Approval will be on hold pending v3_1 release.

**Comment by BenRKarl** [comment]
Once this is merged let's remove the wiki, since it also has a bunch of redundant information. It's removed via Settings.

File: README.rst [hunk 2]
```diff
@@ -7,219 +7,26 @@ Build Status
...
 Installation
-############
-
-This library is distributed via PyPI. If you have not already done so, install
-`pip`_, the following command to install this client library:
-
+------------
 .. code-block::
 
   pip install google-ads
```
Comment (wihl) [inline README.rst:18]: do we even need this line when there is more detailed install info coming to devsite?

Comment (BenRKarl) [reply README.rst:18]: We decided to leave quick reference info like this here for people who just want to know the installation command. I had initially taken it out but put it back in after talking with @mcloonan

**Pull request merged**